key-writer/keywriter.sh

```diff
@@ -159,15 +161,15 @@ BOOTCODE_BINARY_IMAGE=
 BOOTCODE_FLASHING_NAME=
 case ${RPI_DEVICE_FAMILY} in
     4)
-        SOURCE_EEPROM_IMAGE="/lib/firmware/raspberrypi/bootloader-2711/latest/pieeprom-2024-05-17.bin"
-        BOOTCODE_BINARY_IMAGE="/var/lib/rpi-sb-provisioner/recovery.bin"
+        SOURCE_EEPROM_IMAGE="/lib/firmware/raspberrypi/bootloader-2711/latest/pieeprom-2024-07-30.bin"
```

Comment:
I've just released pieeprom-2024-09-10.bin for 2712 - that's also a default/automatic release

Reply:
> I've just released pieeprom-2024-09-10.bin for 2712 - that's also a default/automatic release

I only see a 2024-10-09.bin for 2712 - I'm guessing a new renamed release is coming.

Reply:
ACK .... unless I just leave it thirty days and fixup the commit message ;-)